firefox-ios/Client/Telemetry/SearchTelemetry.swift

```diff
@@ -29,11 +30,95 @@ enum SearchEngine: String, CaseIterable {
     case none
 }
 
+// all the events explained: 
+// https://docs.google.com/document/d/1FFWyxU_uNgjJQATeNbNQKRiSpMztDJKsGc09McUqE0A/edit?pli=1
+// QA about the search bar telemetry:
+// https://docs.google.com/document/d/1Cb4oEoGPTmDCPf0DCbEHhYOS1DRKGRjemsrprhuziKY/edit
```

Comment:
Let's maybe add these docs to the ticket (if they're not already added) and just reference that ticket here, instead fo the docs themselves. :)